docs-ref-conceptual/includes/cli-install-linux-apt.md

```diff
@@ -47,7 +47,7 @@ If you prefer a step-by-step installation process, complete the following steps
     ```bash
     sudo mkdir -p /etc/apt/keyrings
     curl -sLS https://packages.microsoft.com/keys/microsoft.asc |
-      sudo gpg --dearmor -o /etc/apt/keyrings/microsoft.gpg
+      sudo gpg --dearmor > /etc/apt/keyrings/microsoft.gpg
```

Comment:
This command doesn't work and result in error:

```
$ curl -sLS https://packages.microsoft.com/keys/microsoft.asc |
  sudo gpg --dearmor > /etc/apt/keyrings/microsoft.gpg
-bash: /etc/apt/keyrings/microsoft.gpg: Permission denied
curl: (23) Failed writing body
```

It should be reverted to the command before https://github.com/MicrosoftDocs/azure-docs-cli/pull/4106:

```
        gpg --dearmor |
        sudo tee /etc/apt/keyrings/microsoft.gpg > /dev/null
```

Reply:
Interestingly, the script uses `>` and the documentation uses `tee`. Why don't they both use the same `tee` command?

Reply:
There is simply no need doing so. `tee` is used because `>` can't be run as `sudo`.